src/AddTodoForm.js

```diff
@@ -1,11 +1,23 @@
-import React from "react";
+import React, { useState } from "react";
 
-function AddTodoForm(props) {
+function AddTodoForm({onAddTodo}) {
+  const [todoTitle, setTodoTitle] = useState()
+
+
+  const handleTitleChange = (event) => {
+    let newTodoTitle = event.target.value
+    setTodoTitle(newTodoTitle)
+  }
 
   const handleAddTodo = (event) => {
     event.preventDefault();
-    let todoTitle = event.target.title.value
-    props.onAddTodo(todoTitle)
+    // let todoTitle = event.target.title.value
```

Comment:
Best to just remove this unneeded code. If you need it later, it will be visible in previous commits.